graphql-server/src/schemas/neo4j.ts

```diff
@@ -182,7 +186,9 @@ function buildResolvers(
       async requests(_source: undefined, args: any) {
```

Comment:
After the frontend makes the `requests` query, this GraphQL custom resolver gets called to handle that query. How it works:
- Use the Neo4j OGM to query the database. The `where` variable is passed along to handle searching. The `sort` is also passed in, but it will only handle sorting of non-nested fields.
- If the sort setting includes a nested field, we'll call `sortArrayByNestedField` to handle that.
- Finally, we'll return paginated data to handle infinite scroll. We'll only return non-paginated data when the user exports.

A similar flow applies to the `patients` query below.

The `cohorts` query follows a lite version of this flow because it currently handles sorting/filtering on the frontend (doesn't make additional requests when sorting/filtering).